src/lua/lua_manager.cpp

```diff
@@ -89,17 +88,22 @@ namespace big
 		});
 	}
 
-	void lua_manager::load_module(const std::string& module_name)
+	void lua_manager::load_module(const std::filesystem::path& module_path)
 	{
 		std::lock_guard guard(m_module_lock);
 
+		const auto module_name = module_path.filename().string();
+
 		const auto id = rage::joaat(module_name);
 
 		for (const auto& module : m_modules)
 			if (module->module_id() == id)
+			{
```

Comment:
If you're going to expand the inner if block to use a bracket, the outer for loop should too.